src/dparse/lexer.d

```diff
@@ -1786,8 +1786,11 @@ private pure nothrow @safe:
  */
 public auto byToken(ubyte[] range)
 {
+    import std.math : nextPow2;
```

Comment:
Please be a bit careful with relying on newly introduced functions (2.071), in particular such trivial ones.